scripts/gen-service-kustomize.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+#
+# Copyright 2022 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+set -ex
+
+# expect that the common.sh is in the same dir as the calling script
+SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+. ${SCRIPTPATH}/common.sh --source-only
+
+if [ -z "$NAMESPACE" ]; then
+  echo "Please set NAMESPACE"; exit 1
+fi
+
+if [ -z "$KIND" ]; then
+      echo "Please set SERVICE"; exit 1
+fi
+
+if [ -z "$SECRET" ]; then
+      echo "Please set SECRET"; exit 1
+fi
+
+if [ -z "$DEPLOY_DIR" ]; then
+      echo "Please set DEPLOY_DIR"; exit 1
+fi
+
+NAME=${KIND,,}
+
+if [ ! -d ${DEPLOY_DIR} ]; then
+      mkdir -p ${DEPLOY_DIR}
+fi
+
+pushd ${DEPLOY_DIR}
+
+cat <<EOF >kustomization.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+patches:
+- patch: |-
+    - op: replace
+      path: /metadata/namespace
```

Comment:
Couldn't we just use kustomize's --namespace option to do this?

Reply:
yes thats possible, let me update it